package.json

```diff
@@ -46,41 +12,64 @@
   "license": "MIT",
   "scripts": {
     "prebuild": "rimraf public",
-    "build": "gatsby build --prefix-paths",
     "develop": "gatsby develop",
-    "lint": "aegir lint && stylelint \"**/*.css\"",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "webpack": "webpack"
+    "build": "gatsby build --prefix-paths",
+    "serve": "gatsby serve",
+    "lint": "aegir lint && stylelint \"**/*.css\""
+  },
+  "dependencies": {
+    "@researchgate/react-intersection-observer": "~0.7.3",
+    "axios": "~0.18.0",
+    "classnames": "^2.2.6",
+    "gatsby": "^2.0.0",
+    "gatsby-plugin-ipfs": "^2.0.2",
+    "gatsby-plugin-postcss": "^2.0.0",
+    "intersection-observer": "~0.5.0",
+    "normalize.css": "^8.0.0",
+    "postcss-import": "^12.0.0",
+    "postcss-preset-moxy": "^2.3.1",
+    "prismjs": "^1.15.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.5.1",
+    "react-dom": "^16.5.1",
+    "react-helmet": "^5.2.0",
+    "react-intl": "^2.7.1",
+    "react-live": "^1.12.0",
+    "react-markdown": "^4.0.3",
+    "react-resize-detector": "^3.1.3",
+    "react-scroll-to-component": "^1.0.2",
+    "react-slick": "~0.23.1",
+    "react-toastify": "^4.4.0",
+    "react-tooltip": "^3.9.0",
+    "react-youtube": "^7.8.0",
+    "service-worker-gateway": "~0.1.7"
   },
   "devDependencies": {
-    "aegir": "ipfs/aegir#feat/add-eslint-overrides-support",
-    "babel-eslint": "^8.2.6",
-    "babel-preset-moxy": "^2.3.3",
-    "browserslist-config-google": "^1.5.0",
-    "eslint": "^4.19.1",
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "aegir": "^17.0.1",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-macros": "^2.4.2",
+    "babel-preset-moxy": "^2.3.5",
+    "eslint": "^5.8.0",
     "eslint-config-aegir": "^1.0.1",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-react": "^7.10.0",
-    "external-svg-sprite-loader": "^2.0.0",
-    "gatsby-plugin-webpack-bundle-analyzer": "~0.1.1",
-    "postcss-reporter": "^5.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-react": "^7.11.1",
+    "external-svg-sprite-loader": "^4.0.0-beta.7",
+    "file-loader": "^2.0.0",
+    "postcss-reporter": "^6.0.0",
     "raw-loader": "~0.5.1",
     "rimraf": "^2.6.2",
-    "serviceworker-webpack-plugin": "~0.2.3",
-    "stylelint": "^9.2.0",
+    "serviceworker-webpack-plugin": "^1.0.1",
+    "stylelint": "^9.7.0",
     "stylelint-config-moxy": "^5.0.0",
     "svg-css-modules-loader": "^1.6.1",
-    "svgo": "^1.0.5",
-    "svgo-loader": "^2.1.0"
+    "svgo-loader": "^2.2.0"
   },
-  "browserslist": [
```

Comment:
Where is this defined now?

Reply:
I did remove it but should have restored it back! Done now.